### DISCUSSION TITLE
errors in module.css

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#5459] [Q&A]
how to fix these errors?

<img width="928" alt="image" src="https://github.com/mantinedev/mantine/assets/83704092/cb41555e-abcd-43f2-9252-df00d816aa33">

**Comment by rtivital** [comment]
Here is the guide from the community

**How can I set up Webstorm to work with Mantine?**

**PostCSS**
- Enable the bundled [PostCSS](https://plugins.jetbrains.com/plugin/8578-postcss) plugin, if not already enabled by default. After installing the postcss dependency, Webstorm will notify you that the plugin can be enabled.
- Enable PostCSS in .css files in *Preferences | Languages & Frameworks | Stylesheets | Dialects* by setting the Project CSS Dialect to `PostCSS`.

**ESLint**
To activate and configure ESLint in Webstorm, follow this Jetbrains help article: <https://www.jetbrains.com/help/webstorm/eslint.html>

I'm not sure if it works, I do not use Webstorm

**Reply by maposia** [reply]
I have same problem and i don't know how to solve this problem(
What you described above it only solves the problem with the built-in functions from postcss. It does not solve the problem with your preset.

**Reply by PaulTondeur** [reply]
This is indeed not solving the problem. Same goes for custom colors and css variables exposed through `CSSVariablesResolver`.

Seems like the tooling is incomplete in this area with usage in Webstorm.

**Reply by SpechtacularDave** [reply]
Also running into this issue. Wish there was a fix.

**Reply by rtivital** [reply]
Turn off CSS validation in WebStorm
